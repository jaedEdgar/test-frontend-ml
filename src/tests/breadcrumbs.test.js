
import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom'
import Breadcrumbs from "../client/components/breadcrumbs"

const list = [
  "Lorem ipsum dolor",
  "Lorem ipsum dolor",
  "Lorem ipsum dolor",
  "Lorem ipsum dolor",
  "Lorem ipsum dolor"
]


test('Breadcrumbs test snapshot ', () => {
  const component = renderer.create(<Breadcrumbs breadcrumbsList={list} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
