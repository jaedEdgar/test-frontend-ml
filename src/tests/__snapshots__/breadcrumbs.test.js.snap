// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs test snapshot  1`] = `
<div
  className="container"
>
  <ul
    className="breadcrumbs"
  >
    <li
      className="breadcrumbs--item"
    >
      Lorem ipsum dolor
    </li>
    <li
      className="breadcrumbs--item"
    >
      Lorem ipsum dolor
    </li>
    <li
      className="breadcrumbs--item"
    >
      Lorem ipsum dolor
    </li>
    <li
      className="breadcrumbs--item"
    >
      Lorem ipsum dolor
    </li>
    <li
      className="breadcrumbs--item"
    >
      Lorem ipsum dolor
    </li>
  </ul>
</div>
`;

exports[`renders learn Breadcrumbs  1`] = `
<div
  className="container"
>
  <ul
    className="breadcrumbs"
  >
    <li
      className="breadcrumbs--item"
    >
      Lorem ipsum dolor
    </li>
    <li
      className="breadcrumbs--item"
    >
      Lorem ipsum dolor
    </li>
    <li
      className="breadcrumbs--item"
    >
      Lorem ipsum dolor
    </li>
    <li
      className="breadcrumbs--item"
    >
      Lorem ipsum dolor
    </li>
    <li
      className="breadcrumbs--item"
    >
      Lorem ipsum dolor
    </li>
  </ul>
</div>
`;
